.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Request a feature
```

Comment:
Does it make sense to ask about the timeline/priority/level of impact for the request?

Reply:
That can be too sensitive information to ask to share on public, so i don't want to include this to the template.